app/views/guide_to_collecting_mobile_information/index.html.erb

```diff
@@ -30,16 +30,14 @@
 
     <p class="govuk-body">You also need to <%= govuk_link_to 'explain our privacy policy', guide_to_collecting_mobile_information_privacy_path %> to the account holder.</p>
 
-    <h3 class="govuk-heading-m" id="how-to-save-information">How to save information</h3>
-
-    <p class="govuk-body">Use one of the following template spreadsheets to save the information you’re collecting:</p>
+    <h3 class="govuk-heading-m" id="submit-information">Submit the information you collect</h3>
 
+    <p class="govuk-body">You can submit information by using:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li><%= govuk_link_to 'Excel Spreadsheet', '/collecting-mobile-information/extra-mobile-data-requests-template.xlsx' %></li>
+      <li>the ‘New request’ form on the service, making requests one at a time</li>
+      <li>an <%= govuk_link_to 'Excel Spreadsheet', '/collecting-mobile-information/extra-mobile-data-requests-template.xlsx' %>, complete and upload this spreadsheet to make multiple requests at once</li>
```

Comment:
Suggest we change to:

'which allows you to make multiple requests once you upload to the service'